src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -125,6 +156,27 @@ private static CSharpCompilation MetadataView(CSharpCompilation comp, params Met
             return CreateCompilation("", references: references);
         }
 
+        private static CSharpCompilation RetargetedView(CSharpCompilation comp, params MetadataReference[] references)
+        {
+            var someType = comp.GlobalNamespace.GetMembers().OfType<NamedTypeSymbol>().FirstOrDefault();
+            references = references.Append(comp.ToMetadataReference());
+            var result = CreateCompilation("", references: references, targetFramework: TargetFramework.NetStandardLatest);
+            result.VerifyDiagnostics();
+            var originalCorLib = comp.Assembly.CorLibrary;
+            var newCorLib = result.Assembly.CorLibrary;
+            Assert.NotEqual(originalCorLib, newCorLib);
+
+            if (someType is { })
+            {
+                // confirm that it is retargeted
```

Comment:
>retargeted [](start = 38, length = 10)

retargeting vs. retargeted

Reply:
The same issue with variable names.

---
In reply to: [424738892](https://github.com/dotnet/roslyn/pull/44025#discussion_r424738892) [](ancestors = 424738892)

Reply:
I am using it as an adjective, not as a type name. We have an inconsistency in the compiler today with what verb tense we use (SubstitutedSymbol, etc, vs RetargetingSymbol). We should be consistent, and I will file an issue to rename these to use the same past tense that we use everywhere.

---
In reply to: [424742356](https://github.com/dotnet/roslyn/pull/44025#discussion_r424742356) [](ancestors = 424742356,424738892)

Reply:
> I am using it as an adjective, not as a type name. I will change "is" to "has been" for clarity. We have an inconsistency in the compiler today with what verb tense we use (SubstitutedSymbol, etc, vs RetargetingSymbol). We should be consistent, and I will file an issue to rename these to use the same past tense that we use everywhere.

The names reflecting what the symbols are doing, they are performing retargeting, they are not retargeted. References are retargeted, definitions are not, definitions perform retargeting. 

---
In reply to: [425950644](https://github.com/dotnet/roslyn/pull/44025#discussion_r425950644) [](ancestors = 425950644,424742356,424738892)